### PR TITLE
Fix/1730 fix breadcrumb alignment

### PR DESCRIPTION
**Pull request by mr-nick17**
### What

- Fix bug in safari where the last breadcrumb goes out of alignment ✅ [Breadcrumb misalignment](https://trello.com/c/7sbiBbUD/1730-breadcrumb-misalignment)

**Breaking changes**
Below fixes are breaking changes. A [v1](https://github.com/ONSdigital/sixteens/tree/v1) branch has been created, should it be needed.
- Fix audit issues
- Updated node to LTS `Node.js 18.17.0`
  - This has changed how jQuery loads; I've updated the `dependencies` in `package.json` so that jquery auditing now happens. However jquery is no longer loaded through sixteens. Calling apps are to include the following snippet
```html
		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```

### How to review

Sense check
Audit passes

#### Safari before
saf

#### Safari after
image

### Who can review

Frontend dev

### CODE REVIEW

**Comment by lindenmckenzie** [comment]
This will be a breaking change yes?

**Comment by mr-nick17** [comment]
> This will be a breaking change yes?

Yes for jQuery...

I'll bump the version in `package.json`, make a v1 branch and update the above

**Comment by lindenmckenzie** [comment]
Could add a migration path section to the readme as well please? / how to use this version

**Review by lindenmckenzie — COMMENTED** [review]
Few minor comments

File: ci/audit.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ image_resource:
   type: docker-image
   source:
     repository: node
-    tag: 14.15.5
+    tag: 18.17.0
```
[the remark this answers is not in the document]
Comment (mr-nick17) [reply ci/audit.yml:8]: Nothing like have multiple references of the same thing in a repo 🎉

**Review by lindenmckenzie — APPROVED** [review]

**Pull request merged**